include/dem/pp_linear_force.h

```diff
@@ -229,7 +230,8 @@ class PPLinearForce : public PPContactForce<dim>
     const ArrayView<const double> &particle_two_properties,
     Tensor<1, dim> &               normal_force,
     Tensor<1, dim> &               tangential_force,
-    Tensor<1, dim> &               tangential_torque,
+    Tensor<1, dim> &               particle_one_tangential_torque,
+    Tensor<1, dim> &               particle_two_tangential_torque,
```

Comment:
The torques was not correctly calculated?

Reply:
Two torques should be calculated for the particles in a collision (in case they have different sizes). The model worked fine for mono-dispersed systems that we were using.